tools/github/script.sh

```diff
@@ -7,7 +7,7 @@ export MPL_DIR=`python -c 'import matplotlib; print(matplotlib.get_configdir())'
 mkdir -p $MPL_DIR
 touch $MPL_DIR/matplotlibrc
 
-TEST_ARGS="--doctest-modules --cov=skimage"
+TEST_ARGS="--doctest-modules --cov=skimage --showlocals"
```

Comment:
I'd like to keep this configuration. I think it's very useful to have more context on failed tests. At the same time it doesn't increase the output for passing tests.

Reply:
Sure, that's fine.